bucket/maven.json

```diff
@@ -25,8 +22,5 @@
         "hash": {
             "url": "$url.sha1"
         }
-    },
-    "persist": [
-        "conf"
```

Comment:
> The conf directory does not have to be persisted. It contains only examples and is not used by Maven. Maven uses configuration files in the %USERPROFILE%\.m2 directory.

I believe the `conf` directory is the _global_ configuration directory used by Maven and should be persisted:

![image](https://user-images.githubusercontent.com/4730164/32759439-6f8bb6da-c8b7-11e7-8e51-e42a09211587.png)